[Bug]: uv run poe test, one test doesn't pass

### Brief problem description

one test doesn't pass

### Steps to Reproduce

....

### Expected Result

Expected that all tests will pass

### Actual Result

__________________________________ test_get_item_1 ___________________________________

client = <starlette.testclient.TestClient object at 0x10475dd10>

    def test_get_item_1(client: TestClient):
>       get_task_by_id(
            client=client,
            task_id="lab-02-run-local",
            step_id="lab-02-run-local-venv",
            order="pre",
        )

tests/test_items.py:48: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

client = <starlette.testclient.TestClient object at 0x10475dd10>
task_id = 'lab-02-run-local', step_id = 'lab-02-run-local-venv', order = 'pre'

    def get_task_by_id(client: TestClient, task_id: str, step_id: str, order: str):
        response = client.get(f"/items/item/{task_id}?order={order}")
    
>       assert response.status_code == 200
E       assert 404 == 200
E        +  where 404 = <Response [404 Not Found]>.status_code

tests/test_items.py:35: AssertionError
============================== short test summary info ===============================
FAILED tests/test_items.py::test_get_item_1 - assert 404 == 200
============================ 1 failed, 3 passed in 0.03s =============================

--- a/src/app/services/item_service.py
+++ b/src/app/services/item_service.py
@@ -228,7 +228,7 @@ def get_item_by_id_dfs_iterative(
 
                     for task in lab.tasks:
                         counter += 1
-                        if lab.id == item_id:
+                        if task.id == item_id:
                             return FoundItem(task, counter)
 
                         for step in task.steps:

